L1Trigger/L1TTrackMatch/src/L1TkMuCorrDynamicWindows.cc

```diff
@@ -0,0 +1,401 @@
+#include "L1Trigger/L1TTrackMatch/interface/L1TkMuCorrDynamicWindows.h"
+
+// ROOT includes
+#include "TH1.h"
+#include "TH2.h"
+
+L1TkMuCorrDynamicWindows::L1TkMuCorrDynamicWindows(std::vector<double>& bounds, TFile* fIn_theta, TFile* fIn_phi)
+    : wdws_theta_(bounds.size() - 1, MuMatchWindow()),
+      wdws_phi_(bounds.size() - 1, MuMatchWindow()),
+      wdws_theta_S1_(bounds.size() - 1, MuMatchWindow()),
+      wdws_phi_S1_(bounds.size() - 1, MuMatchWindow()) {
+  set_safety_factor(0.5);
+  set_sf_initialrelax(0.0);
+  set_relaxation_pattern(2.0, 6.0);
+  set_do_relax_factor(true);
+
+  track_qual_presel_ = true;
+
+  nbins_ = bounds.size() - 1;
+  for (double b : bounds)
+    bounds_.push_back(b);
```

Comment:
why not just initialize bounds_ directly from bounds?

Reply:
done